Android/Entrevoisins/app/src/main/java/com/openclassrooms/entrevoisins/ui/neighbour_list/ListNeighbourActivity.java

```diff
@@ -2,18 +2,23 @@
 
 import android.os.Bundle;
 import android.support.design.widget.TabLayout;
+import android.support.v4.app.FragmentActivity;
 import android.support.v4.view.ViewPager;
 import android.support.v7.app.AppCompatActivity;
 import android.support.v7.widget.Toolbar;
-import android.widget.Button;
 
 import com.openclassrooms.entrevoisins.R;
+import com.openclassrooms.entrevoisins.events.AddFavoriteNeighbour;
+import com.openclassrooms.entrevoisins.events.ClickOnNeighbourEvent;
+import com.openclassrooms.entrevoisins.model.Neighbour;
+import com.openclassrooms.entrevoisins.service.NeighbourApiService;
+import com.openclassrooms.entrevoisins.service.NeighbourFavoriteList;
 
 import butterknife.BindView;
 import butterknife.ButterKnife;
 import butterknife.OnClick;
 
-public class ListNeighbourActivity extends AppCompatActivity {
+public class ListNeighbourActivity extends AppCompatActivity implements ClickOnNeighbourEvent, AddFavoriteNeighbour {
```

Comment:
Passer `this` en paramètre de `new ListNeighbourPagerAdapter(...);` en tant que ClickOnNeighbourEvent

Reply:
Pour eviter d'instancier une nouvelle activité dans l'adapter

Reply:
Ok